### PR TITLE
Prepare for 5.0

### PR DESCRIPTION
**Pull request by sdressler**

### CODE REVIEW

**Review by jukoson — APPROVED** [review]
// ignore this comment

**Comment by sdressler** [comment]
> Do we also want to change schemas / schema-names ?

Uh, I don't follow?

**Review by oferz — COMMENTED** [review]
I thought `license/s64da.license` was there as a place holder for users to know where to put the license file.

**Comment by sdressler** [comment]
> I thought `license/s64da.license` was there as a place holder for users to know where to put the license file.

Something missing?

**Comment by oferz** [comment]
> > I thought `license/s64da.license` was there as a place holder for users to know where to put the license file.
> 
> Something missing?

I think it was removed in this PR. I don't mind so much, I just thought it was there for a reason. If I'm wrong about this, then all is fine.

**Comment by sdressler** [comment]
> I think it was removed in this PR. I don't mind so much, I just thought it was there for a reason. If I'm wrong about this, then all is fine.

I re-added it. Had some problem with my FS not being case-sensitive.

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Once the instance is up and running you can connect through a psql client or run
 a benchmark using the [Swarm64 DA Benchmark Toolkit](https://github.com/swarm64/s64da-benchmark-toolkit).
 
 Important notice: In order to guarantee compatibility between S64 DA and
-s64da-compose, please checkout the GIT Tag that corresponds to your version of S64 DA. 
+s64da-compose, please checkout the GIT Tag that corresponds to your version of S64 DA.
 For example, if your version of S64 DA is 4.2.0, clone this repository and execute
 `git checkout v4.2.0` within the the repository root folder before proceeding.
```
Comment (oferz) [inline README.md]: Shouldn't `4.2.0` be changed here to `5.0.0`?

File: docker-compose-base.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+version: '3'
```
Comment (oferz) [inline docker-compose-base.yml:1]: How is this base file used? I don't see it included in the other files.

Comment (sdressler) [reply docker-compose-base.yml:1]: As stated in the README, you concatenate multiple compose files, for instance: `docker-compose -f docker-compose-base.yml -f docker-compose-s64da-cpu.yml up`.

File: README.md (hunk 1 above)
Comment (sdressler) [reply README.md]: Yeah, makes sense.

File: docker-compose-base.yml (hunk 2 above)
Comment (oferz) [reply docker-compose-base.yml:1]: Oh, sorry, I missed that.

**Comment by sdressler** [comment]
FYI: we keep this open for now and move it to done before beta.

**Review by jukoson — CHANGES_REQUESTED** [review]
According to our next release, version should be 5.0.0-beta.
Please change all version occurrences.

**Pull request merged**